app/interactors/importer.rb

```diff
@@ -22,100 +16,87 @@ def self.import_all(dir)
+          format_dates! obj, 'date'
+          pbar.increment
+        end
 
-      CalendarDate.insert_many imported_data
+        CalendarDate.insert_many imported_data
+      end
     end
   end
 
   def calendar
     progress_bar(:calendar) do |pbar|
-      imported_data = clean_csv(:calendar) do |obj|
-        timestamps! obj
-        format_dates! obj, 'start_date', 'end_date'
-        pbar.increment
-      end
+      clean_csv(:calendar) do |imported_data|
+        imported_data.each do |obj|
+          format_dates! obj, 'start_date', 'end_date'
+          pbar.increment
+        end
 
-      Calendar.insert_many imported_data
+        Calendar.insert_many imported_data
+      end
     end
   end
 
   def routes
-    progress_bar(:routes) do |pbar|
-      imported_data = clean_csv(:routes) do |obj|
-        timestamps! obj
-        pbar.increment
-      end
-
-      Route.insert_many imported_data
-    end
+    clean_csv(:routes) { |imported_data| Route.insert_many imported_data }
+    puts "Imported Routes"
   end
 
   def shapes
     progress_bar(:shapes) do |pbar|
-      imported_data = clean_csv(:shapes) do |obj|
-        timestamps! obj
-        gis_point! obj, 'shape_pt_lat', 'shape_pt_lon', to: 'shape_pt_latlon'
-        pbar.increment
-      end
+      clean_csv(:shapes) do |imported_data|
+        imported_data.each do |obj|
+          gis_point! obj, 'shape_pt_lat', 'shape_pt_lon', to: 'shape_pt_latlon'
+          pbar.increment
+        end
 
-      Shape.insert_many imported_data
+        Shape.insert_many imported_data
+      end
     end
   end
 
   def stop_times
     progress_bar(:stop_times) do |pbar|
-      imported_data = clean_csv(:stop_times) do |obj|
-        timestamps! obj
-        format_times! obj, 'arrival_time', 'departure_time'
-        pbar.increment
-      end
+      clean_csv(:stop_times) do |imported_data|
+        imported_data.each do |h|
+          format_times! h, 'arrival_time', 'departure_time'
+          pbar.increment
+        end
 
-      StopTime.insert_many imported_data
+        StopTime.insert_many imported_data
+      end
     end
   end
 
   def stops
     progress_bar(:stops) do |pbar|
-      imported_data = clean_csv(:stops) do |obj|
-        timestamps! obj
-        gis_point! obj, 'stop_lat', 'stop_lon', to: 'stop_latlon'
-        pbar.increment
-      end
+      clean_csv(:stops) do |imported_data|
+        imported_data.each do |obj|
+          gis_point! obj, 'stop_lat', 'stop_lon', to: 'stop_latlon'
+          pbar.increment
+        end
 
-      Stop.insert_many imported_data
+        Stop.insert_many imported_data
+      end
     end
   end
 
   def trips
-    progress_bar(:trips) do |pbar|
-      imported_data = clean_csv(:trips) do |obj|
-        timestamps! obj
-        pbar.increment
-      end
-
-      Trip.insert_many imported_data
-    end
+    clean_csv(:trips) { |imported_data| Trip.insert_many imported_data }
+    puts "Imported Trips"
```

Comment:
I added `puts` to the ones that didn't need to use the progress bar anymore (by that I mean _I_ didn't know how to use progress bar when it was a single insert 😄 )